cropper/app/lib/CropStore.scala

```diff
@@ -44,6 +46,11 @@ class CropStore(config: CropperConfig) extends S3ImageStorage(config) {
     }
   }
 
+  private def getOrElseOrNone(theMap: Map[String, String], preferredKey: String, fallbackKey: String): Option[String] = Try(
+    // Return the `preferredKey` value in `theMap` or the `fallbackKey` or `None`
+    theMap.getOrElse(preferredKey, theMap(fallbackKey))
+  ).toOption
```

Comment:
Using Try is a weird pattern here. In essence it is throwing and catching and processing an exception which seems to be overkill and confusing. This seems a lot clearer to me.

```suggestion
  private def getOrElseOrNone(theMap: Map[String, String], preferredKey: String, fallbackKey: String): Option[String] = {
    // Return the `preferredKey` value in `theMap` or the `fallbackKey` or `None`
    theMap.get(preferredKey).orElse(theMap.get(fallbackKey))
  }
```

Reply:
It seems barely longer than the call to this function so it's tempting to put this inline.